src/main/java/mate/controller/car/GetMyCurrentCarsController.java

```diff
@@ -0,0 +1,31 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+
+@WebServlet(urlPatterns = "/getCars")
```

Comment:
Повернися до попереднього чекліста, там були приклади гарних і не дуже URL
```suggestion
@WebServlet(urlPatterns = "/сars/my")
```

Reply:
fixed